src/main/java/mate.academy/service/FileReaderService.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 
+@Component
```

Comment:
The `@Component` annotation is applied to an interface. Typically, annotations like `@Component` are used on concrete classes rather than interfaces. Consider moving this annotation to the implementing class.